src/atoms/slider/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface SliderProps {
+  value: number;
+  onInput: (e: any) => void;
```

Comment:
there is actually a type for react events for example something like `ChangeEvent<HTMLInputElement>` you just need to find out the correct type for range input type